### ISSUE TITLE
Add open issues count on extra pin cards

### ISSUE THREAD

**Issue by guibranco** [anuraghazra/github-readme-stats#2486]
### Is your feature request related to a problem? Please describe.

In the `extra pin` card, add an icon for open issues in that repository

### Describe the solution you'd like

Currently: [![Readme Card](https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=github-readme-stats)](https://github.com/anuraghazra/github-readme-stats)

Example: ![image](https://user-images.githubusercontent.com/3362854/215346183-1b5d2833-dd36-42de-bb4d-1536f8ca54cf.png)

### Additional context

The quantity of open issues is already present in the repositories API response. No extra calls need to be done.

**Comment by rickstaa** [comment]
@guibranco I like the idea. We could also add the open pull request. Feel free to create a PR and I will review it 👍.

**Comment by guibranco** [comment]
> @guibranco I like the idea. We could also add the open pull request. Feel free to create a PR and I will review it 👍.

The API only returns the sum of both data (Issues + Pull Requests)...I can't split it with the existing API schema.

One solution would be requesting the `/pulls` endpoint while it returns data, and counting objects, this could be a expensive operation since the API limit to 100 results per page, so for example, this repository would need 2 more requests to know the exactly quantity of PRs

**Comment by guibranco** [comment]
@rickstaa I've submit the PR, from my side I think is all done, could you please check and let me know if I missed something ?

**Comment by guibranco** [comment]
> We can add issue and badges in readme using shield.io.

I don't get the context ? What you mean ?

**Comment by guibranco** [comment]
> I mean in the repo readme.md file we can add badges of issues, pull requests, contributer counter.

@Aditya-creator-bot yes we can, but what does that have to do with this issue ? You understand what we are talking here ? We are talking about the card, not the readme file, I still don't get your point...

**Comment by guibranco** [comment]
@Aditya-creator-bot you understand why this repository exists ? It generates a card with relevant information of a repository without the person looking to the card needs to open the repository to check this information. 

The person who is looking this card won't open readme file / repository home page to check this anyway, so if we have this in the card, we have more value added to it.

Just to give you real life examples:

 - I have a page where I monitor/track almost 50 cards of this. So, in one quick single look, I can check if any of them has open issues, open pull requests, and also prioritize which one I'll work first based on stars count without opening each one by one to check...

- In my [portfolio/resume page](https://guibranco.github.io/#personal-projects) I also show some cards, with some projects that I created, and with that I can only use one single image instead of a lot of badges.io requests to show all relevant information.

In the below image, you can see 10 cards, and that is just 10 HTTP requests. If I add this information through [badges.io](https://badges.io), I'll have at least 4x more requests (1 request for each information per card), so we are moving from 50 HTTP requests to just 10...
![image](https://user-images.githubusercontent.com/3362854/216591076-ded0181e-75ff-4499-8dae-d134eaec3966.png)

**Comment by rickstaa** [comment]
> @Aditya-creator-bot you understand why this repository exists ? It generates a card with relevant information of a repository without the person looking to the card needs to open the repository to check this information.
> 
> The person who is looking this card won't open readme file / repository home page to check this anyway, so if we have this in the card, we have more value added to it.
> 
> Just to give you real life examples:
> 
> * I have a page where I monitor/track almost 50 cards of this. So, in one quick single look, I can check if any of them has open issues, open pull requests, and also prioritize which one I'll work first based on stars count without opening each one by one to check...
> * In my [portfolio/resume page](https://guibranco.github.io/#personal-projects) I also show some cards, with some projects that I created, and with that I can only use one single image instead of a lot of badges.io requests to show all relevant information.
> 
> In the below image, you can see 10 cards, and that is just 10 HTTP requests. If I add this information through [badges.io](https://badges.io), I'll have at least 4x more requests (1 request for each information per card), so we are moving from 50 HTTP requests to just 10... ![image](https://user-images.githubusercontent.com/3362854/216591076-ded0181e-75ff-4499-8dae-d134eaec3966.png)

Thanks for the lengthy justification of why we should merge #2487 into the main branch; very nice use-case. I review it when I find the time. 👍
